### DISCUSSION TITLE
Add Turn Overview

### DISCUSSION THREAD

**Discussion by TheAdventurer64** [organicmaps/organicmaps#1980] [Ideas]
Adding the ability to view turn directions ahead of time would be a great addition to Organic Maps. Examples:![image](https://user-images.githubusercontent.com/50059322/151831783-2e282c7e-f837-41db-b482-923716563a09.jpeg)![image](https://user-images.githubusercontent.com/50059322/151831797-d6ee0264-586b-4367-8e30-081bba5d73ba.png)

**Comment by biodranik** [comment]
Yes, it would be great. Please create an issue.

**Reply by TheAdventurer64** [reply]
See #1985.
